### ISSUE TITLE
ScrapBatchBadResponse, RequestResponse(status_code=404, text='')

### ISSUE THREAD

**Issue by ZohairAbbas** [markowanga/stweet#112]
Is there any issue with the Scrapper? It stopped working all of a sudden and is showing this error now:

st.TweetsByIdRunner(tweets_by_id_task=id_task, raw_data_outputs=[output_print]).run()
File "C:\Python39\lib\site-packages\stweet\tweets_by_ids_runner\tweets_by_id_runner.py", line 45, in run     
    self._execute_next_tweets_request()
File "C:\Python39\lib\site-packages\stweet\tweets_by_ids_runner\tweets_by_id_runner.py", line 79, in _execute_next_tweets_request
    raise ScrapBatchBadResponse(response)
stweet.exceptions.scrap_batch_bad_response.ScrapBatchBadResponse: RequestResponse(status_code=404, text='')

**Comment by Shan-02** [comment]
Yeah, having a similar error. Imagine it has to do with Twitter's most recent attempts to break their site.

**Comment by Polarbear-mail** [comment]
@markowanga Yes, thanks to the author. I'm wondering how to detect which request bodies Twitter has changed, and maybe I need to learn further to contribute to projects I like, again

**Comment by vaidab** [comment]
Same issue.

**Comment by imgreattt** [comment]
Did we get any solution for this?

**Comment by johnstrong1** [comment]
Any News ?
